app/src/views/TipsView.vue

```diff
@@ -0,0 +1,80 @@
+<script setup lang="ts">
+import MemberDetail from '@/components/MemberDetail.vue'
+import { useMembersStore } from '@/stores/member'
+import { ref, watchEffect } from 'vue'
+
+const { members } = useMembersStore()
+const tipAmountPerAddress = ref(0)
+const totalTipAmount = ref(0)
+
+const countTotalTip = () => {
+  totalTipAmount.value = members.length * tipAmountPerAddress.value
+}
+
+const pushTip = () => {
+  // TO DO
+
+  tipAmountPerAddress.value = 0
+  totalTipAmount.value = 0
+}
+
+const sendTip = () => {
+  // TO DO
+
+  tipAmountPerAddress.value = 0
+  totalTipAmount.value = 0
+}
+
+
+watchEffect(() => console.log(totalTipAmount.value))
+</script>
+
+<template>
+  <div class="w-full px-[40px]">
+    <h2 class="text-black text-center my-[40px]">My teams</h2>
+    <div class="flex flex-col w-full gap-[8px]">
```

Comment:
px-[40px], my-[40px], gap-[8px]
 Use the default tailwind uitlity classe. It keep things organized.